ui/src/main/webapp/app.js

```diff
@@ -16,6 +18,62 @@ const theme = createMuiTheme({
   },
 })
 
+const Banner = props => {
+  const { children } = props
+
+  return (
+    <Query query={getUiConfig}>
+      {({ data, loading, error }) => {
+        if (loading) return <p>Loading...</p>
+        if (error) return <p>Error...</p>
+
+        return (
+          <div>
+            <AppBar
+              position='static'
+              style={{
+                fontFamily: 'Sans-serif',
+                color: data.replication.getUiConfig.color,
+                backgroundColor: data.replication.getUiConfig.background,
+                textAlign: 'center',
+              }}
+            >
+              {data.replication.getUiConfig.header}
+            </AppBar>
+            {children}
+            <AppBar
+              position='fixed'
+              style={{
+                fontFamily: 'Sans-serif',
```

Comment:
I don't think we should override the font so that it is consistent with the font of the rest of the app.